packages/aws-cdk/lib/api/cloudformation-deployments.ts

```diff
@@ -244,7 +244,7 @@ export class CloudFormationDeployments {
   /**
    * Replace the {ACCOUNT} and {REGION} placeholders in all strings found in a complex object.
    */
-  private async replaceEnvPlaceholders<A extends { }>(object: A, env: cxapi.Environment): Promise<A> {
+  public async replaceEnvPlaceholders<A extends { }>(object: A, env: cxapi.Environment): Promise<A> {
```

Comment:
I think I'd rather extract this function out and make it take all necessary objects as arguments, than doing it this way around.